README

```diff
@@ -1,3 +1,4 @@
 first line
 second line 
 third line
+fourth line
```

Comment:
lets skip a line and call this the fifth line

Reply:
No way!